cookbooks/aws-parallelcluster-environment/templates/cfn_bootstrap/cookbook_environment.sh.erb

```diff
@@ -0,0 +1,8 @@
+#
+# cookbook_environment.sh:
+#   Setup ParallelCluster environment variables
```

Comment:
Let's be a bit more specific: here we define only the enavrs that are expected to be execute by the cookbook only. Is that right?